### PR TITLE
Add Civo as a packaged Kubeflow distribution

### LINKED ISSUE #3336: Add Civo to Packaged Kubeflow distribution
I recently contributed the Kubeflow distribution to [Civo Cloud](https://www.civo.com/): https://github.com/civo/kubernetes-marketplace/pull/342 . Here's a link to the Kubeflow distribution on Civo: https://www.civo.com/marketplace/Kubeflow

I would love to add this, if this sounds good?

### PR DESCRIPTION
**Pull request by Rishit-dagli**
Closes #3336

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/ok-to-test

**Comment by Rishit-dagli** [comment]
@jbottum could you please take a look at this PR?

**Comment by jbottum** [comment]
@Rishit-dagli thanks for this PR.   I scanned it quickly and I think this material is specific to a the Civo distribution.   Is that correct?

**Comment by Rishit-dagli** [comment]
Yep this is specific to the Civo distribution?

**Comment by jbottum** [comment]
ok, then this should probably go on the Civo distribution website.

**Comment by Rishit-dagli** [comment]
> ok, then this should probably go on the Civo distribution website.

I was wondering about the process to add a distribution isn't this PR enough to do that?

**Comment by jbottum** [comment]
Hey Rishit - I will let other chime in, @thesuperzapper , but IMO, Civo can be added to the list of supported distributions as defined in, [8c9e5fe](https://github.com/kubeflow/website/pull/3367/commits/8c9e5fe6137221afecd66dea92a31db3e71c490d).   As for the PR portion to  "Add civo installation documentation", that documentation needs to be hosted on the Civo site and not on Kubeflow.org.

File: content/en/docs/distributions/civo/install-kubeflow.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                    
++++
+
+Through this tutorial, I will show you how you can leverage Civo marketplace to single click deploy Kubeflow or using the Civo CLI to deploy Kubeflow on Civo Cloud.
+
+## Instakll Kubeflow using the UI
+
+You can also use an existing cluster you have in the same way but for this article, I will walk you through creating a new cluster and installing Kubeflow.
+
+You can easily create a new cluster from the Civo dashboard. In this example I created a cluster with 4 nodes each of the size "Stanndard Large".
```
Comment (jbottum) [inline content/en/docs/distributions/civo/install-kubeflow.md:14]: Standard has a typo

**Comment by thesuperzapper** [comment]
Holding until https://github.com/kubeflow/website/pull/3381 has been merged.

Note, @Rishit-dagli, you will also need to do a slight update to this PR so the table structure is in line with the new changes.

Also, we are going to be __removing all distribution docs from the website__ soon (see https://github.com/kubeflow/website/issues/3074), so please only include (at MOST) a single-page like the ["Kubeflow on AWS" page](https://www.kubeflow.org/docs/distributions/aws/).

/hold

**Comment by thesuperzapper** [comment]
@Rishit-dagli if Civo is still interested in being listed on the "Installing Kubeflow" table, please raise a new PR that conforms to the new table format. 

If you need an example, here is the PR which added [deployKF](https://github.com/deployKF/deployKF) to the table: https://github.com/kubeflow/website/pull/3542

Also, as @jbottum was saying, you will only be able to list the distribution in the table, we will not be accepting new documentation per https://github.com/kubeflow/website/issues/3074.

/close

**Pull request closed without merging**